.env.j2

```diff
@@ -0,0 +1,23 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+FLASK_ENV=production
+
+# TRELLO REST API
+# Key to authenticate with the Trello REST API: https://developer.atlassian.com/cloud/trello/guides/rest-api/authorization/
+TRELLO_API_KEY={{ trello_api_key }}
+# Token to authenticate with the Trello REST API: https://developer.atlassian.com/cloud/trello/guides/rest-api/authorization/
+TRELLO_API_TOKEN={{ trello_api_token }}
+# ID of Trello board that stores the app's to-do items
+TRELLO_BOARD_ID=65d5be7151463a1d4861d8f0
+# ID of the 'To Do' column on the Trello board that stores the app's to-do items
+TRELLO_TO_DO_COLUMN_LIST_ID=65d5be7151463a1d4861d8f7
+# ID of the 'Done' column on the Trello board that stores the app's to-do items
+TRELLO_DONE_COLUMN_LIST_ID=65d5be7151463a1d4861d8f9
```

Comment:
Very minor - I'd do the same for this exercise but if this were a real project I'd recommend pulling these out into variables in your playbook, even if we keep the values saved inside that as a default. They're not sensitive so we don't need to hide them, but we would likely want to eventually swap them out for a new environment
(Wouldn't bother changing here, but worth noting)